### ISSUE TITLE
L5.4 Error when editing Pages/Blogs with empty content

### ISSUE THREAD

**Issue by od3n** [GrafiteInc/CMS#50]
```
ErrorException in HtmlGenerator.php line 41:
Method Yab\Quarx\Services\Normalizer::__toString() must return a string value (View: /Users/Od3n/Documents/Git/quarx/resources/views/vendor/quarx/modules/blogs/edit.blade.php)
```

**Comment by od3n** [comment]
Updated __toString() method in Yab\Quarx\Services\Normalizer class fix the error.

```
public function __toString()
    {
        if (is_null($this->value)) {
            return 'NULL';
        }

        return $this->value;
    }

```

**Comment by mlantz** [comment]
I'm thinking it would be nicer to have `return '';` rather than inject a NULL string in there.

**Comment by od3n** [comment]
@mlantz yes thank you. but i'm not sure if there's a better way to fix this issue

**Comment by od3n** [comment]
Just notice that the fix already there from #44 pull request. Sorry

**Comment by mlantz** [comment]
Looks like I still need to do a release update, a couple PRs ended up in master not in develop thusly theyre not in the releases :( give me some time today to update that
